src/stories/Jobs/CanPostJob/tests.spec.js

```diff
@@ -0,0 +1,80 @@
+const debugLogger = requireUtil("debugLogger");
+const knex = requireKnex();
+const randomUser = requireUtil("randomUser");
+const contextClassRef = requireUtil("contextHelper");
+
+describe("Test Handler UserCanPostJob", () => {
+  it("dummy_story_which_will_pass", async () => {
```

Comment:
this story is incorrect.
Let me know if its functional or Work in progress,

Reply:
Its old stories and I haven't changed yet, need to change accordingly, It is in WIP